# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/moz.build
# Commit: 496ec0c5a60f
# Full Hash: 496ec0c5a60fdb86b233d036dd4747c477fc33ca
# Author: Dragana Damjanovic <dd.mozilla@gmail.com>
# Date: 2019-10-31 09:53:09
# Regressor Bug: 1581637
# File Overlap Count: 1
# Description:
#   Bug 1581637 - Part 8 - Add Http3Session/Http3Stream. r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D46652
# ==============================================================================

diff -r 63b7f1ff1714 -r 496ec0c5a60f security/manager/ssl/moz.build
--- a/security/manager/ssl/moz.build	Wed Oct 30 20:54:21 2019 +0000
+++ b/security/manager/ssl/moz.build	Wed Oct 30 21:09:09 2019 +0000
@@ -81,6 +81,7 @@
     'RootCertificateTelemetryUtils.h',
     'ScopedNSSTypes.h',
     'SharedCertVerifier.h',
+    'SSLServerCertVerification.h',
     'TransportSecurityInfo.h',
 ]
 
